src/components/Layout/index.jsx

```diff
@@ -71,12 +69,11 @@ export default class Layout extends React.Component {
       loadClass,
       backBtn,
       hasSearch,
-      title,
-      noLogin
+      title
     } = this.props;
 
     const { isNormal, isDev, isAdmin } = this.props.user;
-    const hasMenu = (isDev || isAdmin) && !noLogin;
+    const hasMenu = isDev || isAdmin;
```

Comment:
This noLogin is different parameter noLogin， this for judge home detail page has not sideNav . The previous PR noLogin rename isHome.

Reply:
ok, so the name `noLogin` is not right, maybe `noSideNav` is better